lib/pure/asyncdispatch.nim

```diff
@@ -232,6 +232,18 @@ when defined(windows) or defined(nimdoc):
     result.timers.newHeapQueue()
     result.callbacks = initDeque[proc ()](64)
 
+  proc close*(dispatcher: PDispatcher) =
+    ## Closes the Dispatcher, freeing the resources associated with it.
+    ##
+    ## Raises ``ValueError`` if the dispatcher contains registered handles,
+    ## because it indicates lack of appropriate cleanup in the application.
+    if closeHandle(dispatcher.ioPort) == 0:
+      raiseOSError(osLastError())
+    if dispatcher.handles.len != 0:
+      raise newException(ValueError,
+        "The dispatcher has registered handles when being closed. " &
```

Comment:
Minor thing, but I think that this sentence can be removed. The second sentence implies it anyway.